dev/bots/test.dart

```diff
@@ -176,6 +178,55 @@ Future<bq.BigqueryApi> _getBigqueryApi() async {
   }
 }
 
+List<List<String>> _partitionToolTests() {
```

Comment:
This is necessary to split the tests into two, so that we don't run out of memory while collecting coverage. codecov.io can handle merging coverage reports

Reply:
We should capture in a comment or issue what is the root cause of high memory use. If an issue, then add a comment here linking to the issue.

Reply:
Done